content/learn/book/architecture/_index.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Architecture"
+template = "docs.html"
+insert_anchor_links = "right"
+[extra]
+weight = 10
++++
+
+Bevy is a fairly opinionated framework.
```

Comment:
I don't think "opinionated" is the right word here. It's true that it is opinionated, but I don't see how that relates to the next sentence. Perhaps "integrated" or "interdependent" is the word you are looking for?